runtime/parachain/src/lib.rs

```diff
@@ -509,9 +509,12 @@ construct_runtime!(
         MultiFinalityVerifierGenericLike: pallet_mfv::<Instance4>::{
             Pallet, Call, Storage, Config<T, I>
         } = 104,
-        ContractsRegistry: pallet_contracts_registry::{Pallet, Call, Config<T>, Storage, Event<T>} = 105,
-        CircuitPortal: pallet_circuit_portal::{Pallet, Call, Storage, Event<T>} = 106,
-        Circuit: pallet_circuit::{Pallet, Call, Storage, Event<T>} = 107,
+         MultiFinalityVerifierDefault: pallet_mfv::{
```

Comment:
extra space here?

Reply:
cargo fmt was complaining if it left it 🤷‍♂️